### DISCUSSION TITLE
Allow the ability to see per-map leaderboard positions on playlist scores set on lazer.ppy.sh

### DISCUSSION THREAD

**Discussion by Stoppedpuma** [ppy/osu#27090] [Ideas]
Currently it only shows the overall leaderboard positions on older playlists, this change would allow leaderboards on a per map basis to be shown as well.

**Comment by Rekunan** [comment]
I don't think this should be limited to just the ones on lazer.ppy.sh.

**Reply by Stoppedpuma** [reply]
I think you seem to have misunderstood what I meant, this wouldn't limit to only the ones set on lazer.ppy.sh it would allow the ones from lazer,ppy.sh to be visible as well as the current ones since they currently aren't visible from in the client.

**Reply by Rekunan** [reply]
Ahh ok, that def makes more sense xd.
